apps/prairielearn/package.json

```diff
@@ -70,7 +70,7 @@
     "crypto-js": "^4.1.1",
     "csv-parse": "^5.4.0",
     "csvtojson": "^2.0.10",
-    "d3": "^7.8.4",
+    "d3": "^7.8.5",
```

Comment:
I believe this is client-side (though I don't see an issue with updating it).

Reply:
True, it is client-side, though this is an incredibly stable package - like you said, it should be safe to do this.